### PR TITLE
Add how to test section to PR template

### PR DESCRIPTION
**Pull request by raybejjani**
**Summary of changes**:

Add a How to test section to our PR template. I've found this helps people think about how testable new code is but it also gives the reviewer some direction to verify the changes.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -13,3 +13,6 @@ Fixes: #issue-number
 ```release-note
 <!-- Enter the release note text here if needed -->
 ```
+
+**How to test (optional)**
+e.g. Run `make tests`
```
Comment (aalemayhu) [inline .github/pull_request_template.md]: Would also be nice to mention runtime tests.

**Review by aalemayhu — CHANGES_REQUESTED** [review]
Maybe also link to http://cilium.readthedocs.io/en/latest/contributing/#testsuite ?

File: .github/pull_request_template.md (hunk 1 above)
Comment (raybejjani) [reply .github/pull_request_template.md]: done!

**Review by tgraf — APPROVED** [review]

**Comment by raybejjani** [comment]
@scanf added the link

**Pull request merged**